.github/workflows/operate-ci-build-observe-reusable.yml

```diff
@@ -7,6 +7,11 @@ on:
         description: "The branch name to be used for the workflow"
         required: true
         type: string
+      forkCount:
+        description: "The number of VMs to fork in parallel in order to execute the tests"
+        required: false
+        default: 4
+        type: number
```

Comment:
`input.forkCount` is referred to by this workflow but never defined (but defined like this in similar workflows)